configure-ironic.sh

```diff
@@ -123,11 +123,19 @@ EOF
     fi
 fi
 
+add_options_to_j2_config () {
+  if [[ "${IRONIC_TLS_SETUP}" == "true" ]]
```

Comment:
you should probably not use IRONIC_TLS_SETUP here. you're binding the database TLS setup to the existence of the API or RPC certificates. you should instead make a separate variable for Mariadb, based on the mariadb certificates